src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -26,6 +26,14 @@
       attach="#filing-history-list"
     />
 
+    <!-- Display for if a court order(s) have been filed -->
+    <v-card v-if="hasCourtOrders" class="my-6 pa-6" elevation="0">
```

Comment:
Do you need to set elevation?

Reply:
OK! I don't remember using this before but I guess you need it to match the UI design.